web/controllers/skill_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CodeCorps.SkillController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Skill
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create]
+
+  def index(conn, params) do
+    skills =
+      case params do
+        %{"filter" => filter} ->
+          ids = String.split(filter, ",") |> Enum.map(fn(n) -> String.to_integer(n) end)
```

Comment:
Could add a method to do these steps to break it out for clarity, but not sure how else we are going to convert a string of csv's to a list of numbers...